dali/operators/image/resize/random_resized_crop.h

```diff
@@ -24,21 +24,24 @@
 #include "dali/pipeline/operator/op_spec.h"
 #include "dali/pipeline/operator/common.h"
 #include "dali/operators/image/resize/resize_base.h"
+#include "dali/operators/image/resize/resize_attr.h"
 #include "dali/operators/image/crop/random_crop_attr.h"
 #include "dali/kernels/imgproc/resample/params.h"
 
 namespace dali {
 
 template <typename Backend>
 class RandomResizedCrop : public Operator<Backend>
-                        , protected ResizeBase
-                        , protected RandomCropAttr {
+                        , protected ResamplingFilterAttr
```

Comment:
Do we need to inherit from ResamplingFilterAttr?

Reply:
No, we can have it via composition. Other operators are already refactored.